packages/ui/src/components/Menu/__stories__/Template.stories.tsx

```diff
@@ -1,9 +1,9 @@
 import type { ComponentStory } from '@storybook/react'
 import { Menu } from '..'
-import { Button } from '../../Button'
+import { Button } from '../../index'
 
 export const DefaultDisclosure = (
-  <Button action icon="dots-horizontal" iconSize={24} variant="transparent" />
+  <Button icon="dots-horizontal" variant="ghost" size="medium" />
```

Comment:
```suggestion
  <Button icon="dots-horizontal" variant="ghost" size="small" />
```

Old disclosure button was 32x32, so it should be small size